source/extensions/transport_sockets/tls/cert_validator/default_validator.cc

```diff
@@ -458,8 +459,9 @@ void DefaultCertValidator::addClientValidationContext(SSL_CTX* ctx, bool require
     // Note that BoringSSL call only returns 0 or 1.
     // OpenSSL can also return -1, for example on sk_find calls in an empty list
     if (sk_X509_NAME_find(list.get(), nullptr, name) == 1) {
-      continue;
-    }
+                continue;
+      }
+
```

Comment:
Let's try to keep the changes minimum as they make it harder to sync with upstream later. So, undo this please.